### DISCUSSION TITLE
Redis alternatives => Redis change license

### DISCUSSION THREAD

**Discussion by MaximeWewer** [goauthentik/authentik#11027] [Ideas]
Hi,

With Redis license change https://github.com/redis/redis/pull/13157 it could be necessary to inspect impact of this change and bench alternatives. Ex: https://github.com/dragonflydb/dragonfly or https://github.com/Snapchat/KeyDB.

**Comment by zuiderkwast** [comment]
https://github.com/valkey-io/valkey

**Comment by jmaibaum** [comment]
https://codeberg.org/redict/redict

**Comment by dami-i** [comment]
https://valkey.io/

**Comment by BeryJu** [comment]
As we're not re-selling redis as a service, this license change does not affect us. Anything that is 100% redis compatible should work with authentik just fine, however we do officially only support Redis.

**Reply by madolson** [reply]
What would it take to officially support Valkey in addition to Redis? We might be able to offer testing and engineering help to help setup compatibility testing.

**Reply by larstobi** [reply]
There is this: https://github.com/django-commons/django-valkey

It's a fork of django-redis.

**Comment by dewi-tik** [comment]
Redis is being removed in our 2025.10 release and therefore an alternative isn't required.
